docs/cel2/fee-currencies.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Fee Currencies on Cel2
+---
+
+## Fee Currency Addresses
+
+```bash
+celocli network:whitelist --node https://forno.dango.celo-testnet.org
```

Comment:
Should we also add celocli transfer with -gasCurrency 0x874069Fa1Eb16D44d622F2e0Ca25eeA172369bC1 example ?